docs/tutorial/writing-a-more-complex-dataset-definition/index.md

```diff
@@ -254,14 +254,31 @@ dataset.sex = patients.sex
 ```
 
 We call `patients.age_on` to determine the age of a patient on the index date
-and assign the result column to `dataset.age`.
+and assign the result column to the `age` variable.
 Remember that to be included in the population,
 a patient was an adult on the index date.
 
 ```python
-dataset.age = patients.age_on(index_date)
+age = patients.age_on(index_date)
 ```
 
+We then use the `age` variable to match the patient to one of five age
+bands using `case()` expression and assign this to `dataset.age_band`.
```

Comment:
missing "the" before `case()`?

Reply:
Missing "a" I think, but good catch – thanks.